server/src/main/java/io/pravega/schemaregistry/storage/client/TableStore.java

```diff
@@ -183,12 +183,14 @@ protected void doStop() {
     }
 
     public CompletableFuture<List<Version>> updateEntries(String tableName, Map<byte[], VersionedRecord<byte[]>> batch) {
+        log.info("Update entries called");
```

Comment:
Why is this an INFO log. Why do customers need to know these inner implementation details?

Reply:
Updated with correct label